src/ORM/OrmV3PersistenceStrategy.php

```diff
@@ -47,8 +48,9 @@ public function relationshipMetadata(string $parent, string $child, string $fiel
         }
 
         return new RelationshipMetadata(
-            isCascadePersist: $association->isCascadePersist(),
+            isCascadePersist: ($inversedAssociation ?? $association)->isCascadePersist(),
             inverseField: $metadata->isSingleValuedAssociation($association->fieldName) ? $association->fieldName : null,
+            isCollection: ($inversedAssociation ?? $association) instanceof ToManyAssociationMapping
```

Comment:
I'm not sure about those... but we want to know if the "inversed" association is a collection here, if it exsts. It's the only way to know that we are handling an "inverse one to one"